src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -228,10 +228,9 @@ public bool TryCancel()
 
                         case State.Cancelled:
                             // Someone else cancelled the operation.
-                            // Just return true to indicate the operation was cancelled.
                             // The previous canceller will have fired the completion, etc.
                             Trace("Exit, previously cancelled");
-                            return true;
+                            return false;
```

Comment:
Where is this result used?  I'm wondering why TryCancel doesn't just return `void`.

Reply:
I see, you added a use for it below.

Reply:
Can you add a descriptive comment to TryCancel indicating what the return value means?